airflow/operators/dagrun_operator.py

```diff
@@ -49,6 +50,8 @@ class TriggerDagRunOperator(BaseOperator):
     :type conf: dict
     :param execution_date: Execution date for the dag (templated)
     :type execution_date: str or datetime.datetime
+    :param reset_dag_run: whether or not clear existing dag run if already exists
```

Comment:
Should we add note here that this may be useful for backfill and rerunning tasks?

Reply:
added more description.